### PR TITLE
[WIP] Feature: Add dropdown to choose encryption type

### PR DESCRIPTION
**Pull request by hoailinh1210**
### Summary (Please recap what you changed or fixed)
- Add dropdown and get encryption type from "casper-storage"

image

### Checklist (If you won't pass this checklist please comment why)

-   [x] I removed all commented or unnecessary code.
-   [x] Provide the screenshots due to UI changed or bug fixed
-   [ ] My code has covered these test cases (please list down your tested cases):

### CODE REVIEW

File: src/screens/authentication/create_new_wallet/RecoveryPhraseScreen/index.tsx [hunk 1]
```diff
@@ -54,6 +67,37 @@ const RecoveryPhraseScreen = () => {
     <CLayout>
       <CHeader title={'Recovery Phrase'} />
       <View style={styles.container}>
+        <Row.LR pt={16} px={16}>
+          <View style={styles.selectType}>
+            <Text style={styles.algorithmLabel}>Select Algorithm</Text>
```
Comment (gaupoit) [inline src/screens/authentication/create_new_wallet/RecoveryPhraseScreen/index.tsx:72]: Please change the label to "Choose Encryption Type"

**Review by djhahe — APPROVED** [review]

File: src/screens/authentication/create_new_wallet/RecoveryPhraseScreen/index.tsx (hunk 1 above)
Comment (gaupoit) [reply src/screens/authentication/create_new_wallet/RecoveryPhraseScreen/index.tsx:72]: hello @hoailinh1210 , please have a look. Thanks.

Comment (codsay) [reply src/screens/authentication/create_new_wallet/RecoveryPhraseScreen/index.tsx:72]: @gaupoit @hoailinh1210 we also need to have a small text to explain differences between 2 types

> We recommend to choose ed25519 over secp256k1 for stronger security and better performance, unless you explicitly want to use secp256k1 in order to compatible with Bitcoin, Ethereum chains

Comment (gaupoit) [reply src/screens/authentication/create_new_wallet/RecoveryPhraseScreen/index.tsx:72]: It's valid suggestion. Thanks @codsay . Cc @hoailinh1210 @djhahe .

**Pull request merged**